src/server/pps/server/api_server.go

```diff
@@ -1816,6 +1872,9 @@ func (a *apiServer) validatePipelineRequest(request *pps.CreatePipelineRequest)
 	if request.Spout != nil && request.Autoscaling {
 		return errors.Errorf("autoscaling can't be used with spouts (spouts aren't triggered externally)")
 	}
+	if !request.Autoscaling && request.MaximumExpectedUptime != nil {
+		return errors.Errorf("maximum expected up time cannot be set when autoscaling pipelines is false")
```

Comment:
There is no formatting happening here, so `errors.New` would be more precise than `errors.Errorf`.